client/src/components/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, } from 'react';
+import styles from './ResetPassword.module.scss';
+import TextField from '../TextField/TextField.tsx';
+import { Button } from '@mui/material';
+import logo from '../../assets/Updated_RoadWatch_Logo.svg';
+
+/** 
+ * ResetPassword Component
+ * Represents a form field for resetting password with email input and basic validation.
+ * This component provides a text field for users to enter their email address and displays
+ * an error message if the entered email format is invalid.
+ * 
+*/
+export default function ResetPassword(): JSX.Element {
+    // State to track the validity of the entered email with the format
+    const [isValidEmail, setIsValidEmail] = useState(true);
+
+    /**
+     * handleChange Function
+     * Handles changes in the email input field.
+     * Validates the entered email format.
+     * Updates the isValidEmail state
+     * @param (string) value - the value of the email input field
+     */
+    const handleChange = (value: string) => {
+        // Basic email format validation
+        const isValid = /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(value);
+        setIsValidEmail(isValid);
+    };
+
+    return (
+        <div className={styles['ResetPassword__container']}>
+            <div className={styles['ResetPassword_protectionDiv']} />
+            <div className={styles['ResetPassword__buttonContainer']}>
+                <Button className={styles['ResetPassword__backButton']}>
```

Comment:
I see you've already commented on testing in the 'Things to Update' section, so this just serves as a reminder for later. We need to verify that the back button correctly navigates to the intended page and upon selecting the back button, we should ensure that any text entered in the email field is cleared to avoid leaving any footprints.